back/engines/commercial/verification/spec/acceptance/verification_methods_spec.rb

```diff
@@ -84,13 +91,20 @@
           id: '516e134d-e22b-4386-a783-0db4c2708256',
           type: 'verification_method',
           attributes: {
-            name: 'id_card_lookup',
-            ui_method_name: 'By social security number',
-            card_id: 'Social security number',
-            card_id_tooltip: 'You can find this number on you card. We just check, we don\'t store it',
-            card_id_placeholder: 'xx-xxxxx-xx',
-            explainer_image_url: 'https://some.fake/image.png',
-            action_metadata: { allowed: false }
+            :card_id => 'Social security number',
+            :card_id_placeholder => 'xx-xxxxx-xx',
+            :card_id_tooltip => "You can find this number on you card. We just check, we don't store it",
+            :explainer_image_url => 'https://some.fake/image.png',
+            :method_metadata => {
+              :allowed_for_verified_actions => false,
+              :locked_attributes => [],
+              :locked_custom_fields => [],
+              :name => 'id_card_lookup',
+              :other_attributes => [],
+              :other_custom_fields => []
+            },
+            :name => 'id_card_lookup',
+            :ui_method_name => 'By social security number'
```

Comment:
Would reformat these attributes into `card_id: 'Social security number'` format to match the rest of the object